src/css/_style.scss

```diff
@@ -333,34 +346,77 @@ button {
     flex-direction: row;
     height:25rem;
   }
-
+  
   .sleep-stats {
     grid-row: 5;
     grid-column: 1/3;
   }
-
+  
   .activity-charts {
     grid-column: 1/3;
   }
 }
 
-@media screen and (max-width: 700px) {
+@media screen and (max-width: 760px) {
+  .navigation {
+    grid-template-columns: auto 6rem;
+    grid-template-rows: repeat(4, 4rem);
+    
+    &__drop-down {
```

Comment:
Can we add all borders to dropdown menu display? When the dropdown is open, the choices don't have side borders